plugins/module_utils/network/sonic/config/lldp_interfaces/lldp_interfaces.py

```diff
@@ -80,6 +82,20 @@ def get_lldp_interfaces_facts(self):
             return []
         return lldp_interfaces_facts
 
+    def convert_vlan_list(self, vlan_lst):
+        """
+        Convert a list of VLANs to a new list.
```

Comment:
Please clarify what is being converted.

```suggestion
        Convert a list of strings specifying single VLANs and VLAN
        ranges to a new list containing integer values for single
        vlans and strings for the ranges.
```

Reply:
This comment (and the function it describes) have been modified, but the new comment still does not describe what the list conversion is doing. To reduce confusion, I will mark this "Issue" as "Resolved", but am posting a new one requesting a description that explains the operation performed by the conversion.